asmcomp/s390x/emit.mlp

```diff
@@ -94,6 +100,14 @@ let emit_stack r =
   | _ -> fatal_error "Emit.emit_stack"
 
 
+(* Output a load to a global address *)
```

Comment:
The style `(* Output a ... *)` is more consistent with the other comments of `emit_` functions in the file, but I agree the wording was wrong. I chose the still awkward but more correct "Output a load of the address of a global symbol".